libpod/util.go

```diff
@@ -29,6 +33,87 @@ const (
 	unknownPackage = "Unknown"
 )
 
+// Handle volume options from CLI.
+// Parse "o" option to find UID, GID, Size.
+func VolumeOptions(opts map[string]string) ([]VolumeCreateOption, error) {
+	libpodOptions := []VolumeCreateOption{}
+	volumeOptions := make(map[string]string)
+
+	for key, value := range opts {
+		switch key {
+		case "o":
+			// o has special handling to parse out UID, GID.
+			// These are separate Libpod options.
+			splitVal := strings.Split(value, ",")
+			finalVal := []string{}
+			for _, o := range splitVal {
+				// Options will be formatted as either "opt" or
+				// "opt=value"
+				splitO := strings.SplitN(o, "=", 2)
+				switch strings.ToLower(splitO[0]) {
```

Comment:
There should be a check to make sure len(split0) == 2.

Reply:
This is just a move of exact function from `pkg/domain/infra/abi/parse/parse.go`

Reply:
Which had the bug, so it needs to be fixed.  Can we consolidate the function there?

Reply:
But fixed